### ISSUE TITLE
Another bot build request

### ISSUE THREAD

**Issue by aggregator01** [binary-com/binary-bot#1906]
Hi All,

Yes I'm a newbie and predictably requesting help with a bot build. However, the logic of it is quite simple - I just have no idea how to code it, so here I am requesting help. As a first port of call, if anyone on here uses fiverr or similar I am happy to pay a few dollars for your help - this isn't a complicated build.

**Comment by Adammodex** [comment]
I'm not a coder too but we don't pay to get help here dude, just ask!
let me tag some coders for you.

@binary-ex-machina
@lasaidhari
@Hooponopono
@hotmatrixx
@AlienFly
@murali139
@baruga
@ghost
@Eshhh

**Comment by binary-ex-machina** [comment]
@aggregator01 try to post the strategy here and some users will try to help. In case of no success or if you don't want to lose time and learn how to build the bot, you can try on fiverr, no problem with that. I think most of the people here are against marketing and merchandising here, no one wants to read spam and scam when trying to find help.

**Comment by aggregator01** [comment]
Hi Guys,

Firstly thank you for the kind replies.

My strategy is simple Martingale, however, I wish to have variables, which, I can alter to refine. The basic premise is to balance the losses against the wins. I appreciate, particularly from reading the many posts on here, that this strategy is the usual fabled strategy that all beginners start with. However, from studying the charts I believe that there can be a methodology which will yield profit, albeit small profit against the number of trades required.

I have been playing with the blocky bot programmer so I understand the principal, if someone could help build a bot that can do the below, this will be a good starting point and allow me to understand how the functions work so hopefully I can develop.

As a first iteration I need a bot that can do the following:

1)I wish to chose what market will be traded (I understand how this is set within block 1). It will operate strictly in  RISE / FALL
2)I wish to set the trade time (i.e. 30s, 1 minute etc.)
3)I wish to be able to alter the initial stake 
3)The bot will consider the previous candle and place a trade based on that upon the opening of the new candle. i.e. if the previous minute candle (or 30s candle) was red the bot will place a FALL trade for the next candle and conversely, if the previous minute candle was green the bot will place a RISE trade for the next candle. (If possible I would like to be able to vary this. e.g. if the last candle was red I can choose the bot to place a RISE trade, eventually, I would also like to be able to alter the number of candles considered - initially I will only consider the previous candle then perhaps the previous 2 or the previous 3 etc. but hopefully I can develop this myself)
4) if the trade wins the bot starts again using the initial stake. If the trade loses the bot enters martingale. However, I wish to be able to vary the levels of martingale and the martingale coefficient applied to the initial stake at each level. upon the next winning trade, the bot resets back to the initial stake.


The above explanation is a bit wordy but I hope you can understand.
